source/uwp/Renderer/idl/AdaptiveCards.Uwp.idl

```diff
@@ -1945,6 +1976,28 @@ namespace AdaptiveCards
             [default] interface IAdaptiveRenderArgs;
         };
 
+        [
+            uuid(c0222295-bbec-4ff6-bdf3-f207ce78df69),
+            version(NTDDI_WIN10_RS1)
+        ]
+        interface IAdaptiveFailure : IInspectable
```

Comment:
>IAdaptiveFailure [](start = 18, length = 16)

Still looking for a better name.

Reply:
What if separated this out into warnings vs. errors everywhere?  We could make two separate interfaces/enums/add methods/etc. one for warnings and one for errors.  

---
In reply to: [151170456](https://github.com/Microsoft/AdaptiveCards/pull/950#discussion_r151170456) [](ancestors = 151170456)